src/main/java/com/github/tomakehurst/wiremock/matching/RequestPattern.java

```diff
@@ -139,6 +209,7 @@ public RequestPattern(
       @JsonProperty("urlPathPattern") String urlPathPattern,
       @JsonProperty("urlPathTemplate") String urlPathTemplate,
       @JsonProperty("method") RequestMethod method,
+      @JsonProperty("methods") Set<RequestMethod> methods,
```

Comment:
Hmm. Was the previous constructor supposed to retain binary compatibility?

Reply:
It's better to leave it for now. IMHO